src/services/UserService.ts

```diff
@@ -76,13 +76,42 @@ const getUserPreference = async (payload: any): Promise<any> => {
   });
 }
 
+const updateProductStoreSetting = async (payload: any): Promise<any> => {
+  return api({
+    url: "service/updateProductStoreSetting",
+    method: "post",
+    data: payload
+  });
+}
+
+const createProductStoreSetting = async (payload: any): Promise<any> => {
```

Comment:
```suggestion
const createProductIdentificationPref = async (payload: any): Promise<any> => {
```
Applies to all the other methods